### PR TITLE
DSLAB/klint: add prez

### PR DESCRIPTION
**Pull request by tharvik**
small prez, will probably get updated along the way.

### CODE REVIEW

File: views/products/presentation/klint.tpl [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<p>
+  Klint allows for fast and automated
+  <a href="https://en.wikipedia.org/wiki/Formal_verification"
+    >formal verification</a
+  >
+  of network functions (NF), such as a
+  <a href="https://en.wikipedia.org/wiki/Router_(computing)">router</a> or a
+  <a href="https://en.wikipedia.org/wiki/Network_address_translation">NAT</a>.
+  It works directly on binaries, without the need of special data structure.
```
Comment (ineiti) [inline views/products/presentation/klint.tpl:9]:
```suggestion
  It works directly on binaries, without the need of special data structures.
```

File: views/products/presentation/klint.tpl [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <br />
+  It can ensure that a given NF, be it proprietary of not, respects a
```
Comment (ineiti) [inline views/products/presentation/klint.tpl:11]:
```suggestion
  It can ensure that a given NF, be it proprietary or not, respects a
```

File: views/products/presentation/klint.tpl [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  specification. It allows decoupling between NF-providers and network
```
Comment (ineiti) [inline views/products/presentation/klint.tpl:12]: Add `formal` in front of specification?

**Review by ineiti — CHANGES_REQUESTED** [review]
Some typos. Just for my understanding: the source needs to include a specific library for it to work, correct? If yes, then I would rephrase a bit the description of how it works.

**Comment by tharvik** [comment]
> the source needs to include a specific library for it to work, correct? If yes, then I would rephrase a bit the description of how it works.

hum, yes and no? it needs to implement two functions defined by klint but doesn't need to link w/ it nor to use it as it is then available via DPDK or tinynf implementation.

rephrasing to
>   The network functions can be written in any compiled language, as long as it exposes two functions conforming to Klint's: a setup method taking care of  initialisation, and another method handling a packet, asserting on the transmitted packet.

File: views/products/presentation/klint.tpl [hunk 4]
```diff
@@ -23,10 +23,10 @@
 /></a>
 
 <p>
-  The network functions can be written in any compiled language, exposing a
-  setup method taking care of initialisation, and another method handling a
-  packet. The specification is in regular Python, asserting on the transmitted
-  packet.
+  The network functions can be written in any compiled language, as long as it
+  exposes two functions conforming to Klint's: a setup method taking care of
```
Comment (ineiti) [inline views/products/presentation/klint.tpl:27]:
```suggestion
  exposes two functions conforming to Klint: a setup method taking care of
```

**Review by ineiti — APPROVED** [review]
